src/backends/torch/torchlib.h

```diff
@@ -71,20 +82,22 @@ namespace dd
              TorchDataset &dataset, int batch_size, APIData &out);
 
   public:
-    unsigned int _nclasses = 0;
-    std::string _template;
-    bool _finetuning = false;
-    torch::Device _device = torch::Device("cpu");
-    bool _masked_lm = false;
-    bool _seq_training = false;
-    bool _classification = false;
-    bool _timeserie = false;
-    std::string _loss = "";
-
-    APIData _template_params;
-
-    // models
-    TorchModule _module;
+    unsigned int _nclasses = 0; /**< number of classes*/
+    std::string _template; /**< template identifier (recurrent/bert/gpt2...)*/
+    bool _finetuning = false; /**< add a classif layer after model */
```

Comment:
add a custom head? because with regression #924 it's not a classification layer anymore

Reply:
ok done